README.md

```diff
@@ -579,6 +579,14 @@ class Html5Qrcode {
    */
   constructor(elementId: string, config:  Html5QrcodeFullConfig | undefined);
 
+  /**
+   * Initialize QR Code scanner.
+   * 
+   * @param element - The HTML DOM element.
+   * @param verbose - optional configuration object
+   */
+  constructor(element: HTMLElement, config:  Html5QrcodeFullConfig | undefined);
```

Comment:
Instead of new constructor change the above documentation to 

```ts
/**
   * Initialize QR Code scanner.
   * 
   * @param elementOrElementId - The HTML DOM element or Id of the HTML element.
   * @param verbose - optional configuration object
   */
constructor(elementOrElementId: string | HTMLElement, config: Html5QrcodeFullConfig | undefined);
```